src/plugin.json

```diff
@@ -8,7 +8,10 @@
   "preload": true,
   "dependencies": {
     "grafanaDependency": ">=11.5.0",
-    "plugins": []
+    "plugins": [],
+    "extensions": {
+      "exposedComponents": ["grafana-insights-app/insights-launcher/v1"]
```

Comment:
The plugin uses different name now: `grafana-o11yinsights-app/insights-launcher/v1`